keps/sig-node/213-run-as-group/kep.yaml

```diff
@@ -11,7 +11,12 @@ reviewers:
 approvers:
   - "@liggitt"
   - "@derekwaynecarr"
+prr-approvers:
+  - "@johnbelamaric"
 editor: TBD
 creation-date: 2017-06-21
-last-updated: 2019-02-14
+last-updated: 2021-02-04
 status: implementable
+stage: "stable"
+
+latest-milestone: "v1.21"
```

Comment:
Can you add the `milestone` stanza here, documenting when it went alpha/beta/stable?

Reply:
done